src/Vex/Containers/DirtyFlagged.h

```diff
@@ -0,0 +1,82 @@
+﻿#pragma once
+#include <utility>
+
+namespace vex
+{
+
+template <class T>
+class DirtyFlagged
+{
+    mutable bool dirtyFlag = true;
```

Comment:
We can keep it if you see a practical future use for it. If its just theoretical for now, I'd maybe keep it in a stash somewhere?